aiqleads/scripts/run.py

```
@@ -46,8 +46,12 @@
             status = tracker.get_component_status(path)
             assert status["status"] == "active", f"Unexpected status for {path}"
             
+        # Test path validation
+        assert not tracker._validate_path("../invalid/path.py")
```

Ruff (0.8.2)

Use of `assert` detected

(S101)